api/keyword.go

```diff
@@ -0,0 +1,57 @@
+package api
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/joschahenningsen/TUM-Live/dao"
+	"github.com/joschahenningsen/TUM-Live/model"
+	"github.com/joschahenningsen/TUM-Live/tools"
+	log "github.com/sirupsen/logrus"
+	"net/http"
+	"strconv"
+)
+
+func configGinKeywordRouter(router *gin.Engine, daoWrapper dao.DaoWrapper) {
+	routes := keywordRoutes{daoWrapper}
+	router.Use(tools.Admin) // Todo: Can workers post as admin?
```

Comment:
Workers don't use the http api, they communicate using grpc. See:

https://github.com/joschahenningsen/TUM-Live/blob/dev/worker/api/api.proto
https://github.com/joschahenningsen/TUM-Live/blob/fcc1301ac22ad75feaf78306babd4bfc924106a8/worker/Makefile
https://github.com/joschahenningsen/TUM-Live/blob/dev/api/worker_grpc.go